src/polycap-capil.c

```diff
@@ -486,6 +488,7 @@ int polycap_capil_reflect(polycap_photon *photon, double alfa, polycap_vector3 s
 			}
 			//polycap_capil_trace should be ran description->profile->nmax at most,
 			//which means it essentially reflected once every known capillary coordinate
+			//TODO: make ix_temp more appropriate than 0 (i.e. find ix_temp closer to current interaction)
 			for(i=0; i<=description->profile->nmax; i++){
 				iesc_temp = polycap_capil_trace(ix_temp, phot_temp, description, capx_temp, capy_temp, leak_calc, error);
```

Comment:
Sorry nogal druk voor de moment... volgend week ga ik op verlof voor twee weken en wil nog redelijk wat afwerken.

Het enige dat ik kan zeggen is dat jij altijd moet checken of een functie een probleem heeft gehad: je kan daarvoor niet vertrouwen op `error`, want de gebruiker is niet verplicht dat te gebruiken. Beter dat je daarom altijd een gepaste return value gebruikt die ook aangeeft dat er iets mis is gelopen (en check hiervoor dan!).